do.sh

```diff
@@ -77,6 +123,8 @@ function install_deps() {
     [ -d /var/lib/registry ] || mkdir /var/lib/registry -p
     docker run --privileged -d --name registry -p 5000:5000 \
           -v /var/lib/registry:/var/lib/registry --restart=always docker.io/library/registry:2
+    if [ -d /etc/containers ]
```

Comment:
Do we actually need this at all?  We don't pull anything on the orchestrator node so can this whole block go away?  Of course, not a problem of this commit.

Reply:
Fixed in https://github.com/dceara/ovn-heater/pull/157/commits/c7af0c657747450952e8eb495cad8116fbc80f52

Reply:
It turns out this *IS* required when using podman, otherwise we'll get the following while building the OVS/OVN containers which is done on the orchestrator:
```
Error: trying to reuse blob sha256:178c5a3203aea37107a1afcc9a965d6f9058cb12e31f6ecc900a48c310b27389 at destination: pinging container registry localhost:5000: Get "https://localhost:5000/v2/": http: server gave HTTP response to HTTPS client
```

I guess we can prune it to only hold the `[registries.insecure]` and `[registries.block]` sections?